manim/plugins/import_plugins.py

```diff
@@ -33,9 +33,9 @@
         # it will return a list of modules to add globally
         # finally add it
         lists = loaded_plugin()
-        for l in lists:
-            exec(f"{l.__name__}=l")
-            __all__.append(l.__name__)
+        for x in lists:
+            exec(f"{x.__name__}=x")
+            __all__.append(x.__name__)
```

Comment:
Renaming `l` to `x` is not a particularly great improvement.

```suggestion
        for lst in lists:
            exec(f"{lst.__name__}=lst")
            __all__.append(lst.__name__)
```